Packs/ThreatIntelReports/ReleaseNotes/1_0_22.md

```diff
@@ -0,0 +1,18 @@
+
+#### Layouts
+
+##### Vulnerability Report
+
+- Updated the Vulnerability Report layout. The *Canvas* tab has been removed.
+##### Executive Brief Report
+
+- Updated the Executive Brief Report layout. The *Canvas* tab has been removed.
+##### Threat Actor Report
+
+- Updated the Threat Actor Report layout. The *Canvas* tab has been removed.
+##### Campaign Report
+
+- Updated the Campaign Report layout. The *Canvas* tab has been removed.
+##### Malware Report
+
+- Updated the Malware Report layout. The *Canvas* tab has been removed.
```

Comment:
```suggestion
Updated the Vulnerability Report layout. The *Canvas* tab has been removed.
##### Executive Brief Report

Updated the Executive Brief Report layout. The *Canvas* tab has been removed.
##### Threat Actor Report

Updated the Threat Actor Report layout. The *Canvas* tab has been removed.
##### Campaign Report

Updated the Campaign Report layout. The *Canvas* tab has been removed.
##### Malware Report

Updated the Malware Report layout. The *Canvas* tab has been removed.
```

Reply:
It will not change because it is not related to the above contribution but to the commit from master.